docs/alerting/monitors.md

```diff
@@ -41,6 +41,41 @@ Destination | A reusable location for an action, such as Amazon Chime, Slack, or
 
+###  Email as a destination
+To receive an alert notification as an email, you must set up **Email** as the destination **Type** for the alert, and configure **Sender** email addresses and **Recipients** or email groups. You can do this using **Manage senders** and **Manage email groups**. 
+
+#### Manage senders
+Manage senders allows you to configure and manage **Sender** email addresses. Senders are email accounts from where the alert notification is sent to different recipients or email groups. Unless you configure at least one sender account in the settings, the **Sender** dropdown list will be empty. 
+
+To configure a sender email, do the following.
+
+1. Choose **Manage senders**.
+1. In the **Manage email senders** modal window, choose **Add sender**. 
+1. Next, choose **New sender**. Multiple senders can be added one at a time.
+1. Enter a unique **Sender name**. Duplicates are not allowed.
+1. Enter the **Email address**, SMTP **Host** (e.g. `smtp.gmail.com` for a Gmail account), and the **Port** number.
+1. You can choose to use an **Encryption method** or leave it as **None**. **SSL** or **TLS** are standard protocols to secure your email. To use SSL or TLS encryption, you must enter your secure credentials to verify access to the account i.e. your username and password. You can enter these credentials in the Elasticsearch Keystore using the CLI. Run the following commands (in the bin directory of your Elasticsearch directory) to enter your username and password. The `<sender_name>` is the name you entered for **Sender**.
+    `./bin/elasticsearch-keystore add opendistro.alerting.destination.email.<sender_name>.username` 
+    `./bin/elasticsearch-keystore add opendistro.alerting.destination.email.<sender_name>.password`
+1. Choose **Save** to save the configuration and create the sender.
+
+Once the sender is created, the sender account is available to be selected when creating an email destination, and can be used for multiple destinations. However, a destination only allows one sender. 
+
+#### Manage email groups or recipients 
+You can use **Manage email groups** to create and manage a list of email addresses that you frequently send to different groups. Multiple recipients can be grouped into mailing lists based on the type of alert they should receive. For example, a certain type of alert may be triggered for the DevOps team (one email group) and another for the engineering team (another email group). 
+
+You can enter individual email addresses, or an email group in the **Recipients** field. For email groups, you can pre-create a group using **Manage email groups**.
+
+To create and manage email groups, do the following.
+1. Choose **Manage email groups**, and then **Add email group**.
+1. Choose **New email group**.
```

Comment:
Delete

Reply:
Not sure what you want me to delete here. "Choose New email group" is a necessary step because when you actually use the feature, it gets confusing. There is a tendency to keep clicking on "Add email group" without knowing that you need to click on "New email group"